modules/conversantAnalyticsAdapter.js

```diff
@@ -0,0 +1,564 @@
+import {ajax} from '../src/ajax.js';
+import adapter from '../libraries/analyticsAdapter/AnalyticsAdapter.js';
+import CONSTANTS from '../src/constants.json';
+import {getGlobal} from '../src/prebidGlobal.js';
+import adapterManager from '../src/adapterManager.js';
+import * as utils from '../src/utils.js';
```

Comment:
Better to just import the needed utils instead of all here.

Reply:
Looks to just be a couple log utils, deepAccess, and isInteger. So very small amount of the util lib! So just import those specifically.